### DISCUSSION TITLE
DX Cluster

### DISCUSSION THREAD

**Discussion by aa5sh** [foldynl/QLog#468] [Ideas]
I had a couple thoughts about the DX Cluster.  
1. I know I tweaked the Deduplication feature to try and make it work.  But the way it works now if a spot is a dupe it is just ignored.  I think it may be more useful to replace the spot instead, since the comment may have more relavant/current information (up 2 or whatever).  Have it at a minimum update the time and comment?
2. Add a function similar to the Alerts widget to clear older cluster spots.  This is why I would like the time updated like mentioned above so if a station is on a frequency all day it would stay in the list.
3. Also it would be nice if I work a station that the color for that spot is updated appropriately.

**Comment by foldynl** [comment]
>>> I know I tweaked the Deduplication feature to try and make it work. But the way it works now if a spot is a dupe it is just ignored. I think it may be more useful to replace the spot instead, since the comment may have more relavant/current information (up 2 or whatever). Have it at a minimum update the time and comment?

That's not a bad idea. It's worth considering implementing it. However, I'm not sure about the benefit of this update. Both the information from the first spot and the information from the second spot can be 'incorrect' (example: comment). So, with the update, you might not gain anything more than by ignoring the second one. It will likely lead to more complex source code rather than better operator awareness. Or am I wrong?

>>>Add a function similar to the Alerts widget to clear older cluster spots. This is why I would like the time updated like mentioned above so if a station is on a frequency all day it would stay in the list.

I would like to maintain a linear display of DXC Spots over time. Therefore, I don't think something like this should be implemented in DXC.

>>>Also it would be nice if I work a station that the color for that spot is updated appropriately.

This is implemented in the Bandmap. Unfortunately, due to performance reasons, I didn't include it in the DXC. Keep in mind that some users connect to the RBN. This network generates 10,000 to 100,000 QSOs per hour. If I have to recalculate the QSO status for each record in the DXC after every entered QSO, it wouldn't be very good for performance. Of course, new QSOs are already updated according to the new status, but the old ones remain according to the old status. It would be interesting to try implementing this and then test it during a contest against the RBN to see how it affects CPU load and speed.

**Comment by aa5sh** [comment]
I guess I need to adjust my way of thinking a little.  For me I guess more of what I use the DX Cluster for I just need to watch the Alerts and not pay attention to the DX Cluster as much maybe.  But for that I wish the Alerts showed the membership information.

I do understand about the RBN and that flow.  I guess I never was a big band map user except in the rare occasion I am in a contest, so I haven’t even opened in QLog.

**Comment by TecnoTalarn** [comment]
The DX cluster allows you to filter different things from its own interface, but I really don't know exactly what it does, and in the end I decided not to do any filtering in the DXC itself, and play from the Qlog alerts.
I have a handful of alerts, starting with DXCCs I haven't worked, or interesting countries. I also alert dxpeditions or those slots that interest me. It works pretty well with the data it sees on the DXC while the cluster I look at it less and less.
With the Bandmap in view and the alerts (thanks for integrating them on the main screen) You get a quick idea of ​​what you can do in the band where you are and the alerts warn you if any interesting indicator appears anywhere in the HF.
It might be interesting if, in addition to the predefined colors, you could mark a specific alert, for example, a certain country, which even if you have confirmed it, may be of interest to you.

If the Qlog had a contest mode, nothing else would be needed to tidy up the contacts.

**Reply by rsaue** [reply]
Hi!
Being a regular user of CQRLOG Alpha on Linux due to the SQL server option, I do my spot filtering on the cluster. 

If you are connected to a cluster with CC-server software like the VE7CC cluster, a great way to filter out not wanted entities is to use the SET/FILTER command. 

I filter out most Europeans and some other countries with the command

SET/FILTER DXCTY/REJECT K,VE,4U1I,4U1V,9A,9H,CT,CU,DL,E7,EA,EA6,EI,ER,ES,EU,F,G,GD,GI,GJ,GM,GM/S,GU,GW,HA,HB,HB0,HV,I,IS,IT9,LX,LY,LZ,OE,OH,OH0,OJ0,OK,OM,ON,OY,OZ,PA,S5,SM,SP,SV,SV5,SV9,T7,TA1,TF,TK,UA,UA2,UR,YL,YO,YU,CT3,EA8,EA9,IG9,4X,BY,JY,OD,TA,UA9,LU,PY

73 de LB2EG Richard
